custom-vu/src/main/kotlin/jces1209/vu/JiraCloudScenario.kt

```diff
@@ -53,11 +53,6 @@ class JiraCloudScenario : Scenario {
                 projectMemory = similarities.projectMemory,
                 createIssueButtons = listOf(By.id("createGlobalItem"), By.id("createGlobalItemIconButton"))
             ),
-            searchWithJql = SearchCloudFilter(
-                jira = jira,
-                meter = meter,
-                filters = similarities.filtersMemory
-            ),
```

Comment:
I like how much the scenario differences decrease and scenario similarities increase. It gives us more and more confidence that we're comparing apples to apples 👍